jstests/core/explain_verbosity_unknown.js

```diff
@@ -0,0 +1,22 @@
+//Test whether unknown commands are accepted in explain function
+
+(function() {
+	
+	"use strict";
+	
+	var unknownCollection = db["jstestDB"];
+
+	//insure the collection has an item
+	unknownCollection.insert({a:1});
+
+	//check with finding item that exist
+	assert.commandFailed(db.runCommand({explain: {find: 'a'}, verbosity: 'queryPlanner', unknown: 1}), "unknown argument command extra");
```

Comment:
Sorry for misleading you earlier, but let's use the more-specific `assert.commandFailedWithCode()`: https://github.com/mongodb/mongo/blob/r3.5.3/src/mongo/shell/assert.js#L382

After my comment below, this should fail with the code FailedToParse: https://github.com/mongodb/mongo/blob/r3.5.3/src/mongo/base/error_codes.err

In javascript, you can access this by using `ErrorCodes.FailedToParse`, instead of using the number.